py/tests/swagger_client/apis/external_api.py

```diff
@@ -40,6 +40,214 @@ def __init__(self, api_client=None):
                 config.api_client = ApiClient()
```

Comment:
remove the python files from this PR, they will give problems and are auto generated anyway,